test/regressions/index.test.ts

```diff
@@ -189,7 +188,6 @@ async function main() {
     it('should position the headers matching the columns', async () => {
       const route = '/docs-data-grid-virtualization/ColumnVirtualizationGrid';
       const screenshotPath = path.resolve(screenshotDir, `.${route}ScrollLeft400px.png`);
-      await fse.ensureDir(path.dirname(screenshotPath));
```

Comment:
`emptyDir` already ensures `screenshotDir` is available, so I don't think we need to check again.

It's true that something might have deleted it, but since these are tests they will fail and I suppose that's fine.

Reply:
👍  In any case, `fs.mkdir` with `recursive` option should have the same behavior.